recipes/metabolishmm/meta.yaml

```diff
@@ -43,9 +43,8 @@ test:
 
 about:
   home: https://github.com/elizabethmcd/metabolisHMM/wiki
-  license: GNU General Public v3 (GPLv3)
+  license: GPL3
   license_family: GPL3
-  license_file: LICENSE
```

Comment:
Will the license automatically be included in the Conda package with `license: GPL3`?

Reply:
No, it's not packaged by the author, so we don't bother regardless of the fact that it's technically required.

Reply:
Strange.